app/views/businesses/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<h1>Businesses</h1>
+
+<head>
+<style>
+table {
+  font-family: arial, sans-serif;
+  border-collapse: collapse;
+  width: 100%;
+}
+
+td, th {
+  border: 1px solid #dddddd;
+  text-align: left;
+  padding: 8px;
+}
+
+tr:nth-child(even) {
+  background-color: #dddddd;
+}
+</style>
+</head>
+ 
+<div>
+  <table>
+    <tr>
+      <th>Id</th>
+      <th>Name</th>
+      <th>Link</th>
+      <th>Description</th>
+    </tr>
+  <% @businesses.each do |b| %>
+    <tr>
+      <td><%= b.id %></td>
+      <td><%= b.name %></td>
+      <td><%= b.link %></td>
+      <td><%= b.description %></td>
+    </tr>
+  <% end %>
+</table>  
+</div>
```

Comment:
This view is temporary, and just here so that our internal team can check the businesses they have added. The end goal is to have businesses displayed in the same way we display projects